api/src/main/resources/liquibase.xml

```diff
@@ -195,6 +195,16 @@
         </sql>
     </changeSet>
 
+    <changeSet id="create-column-teleconsultation-202008241355" author="Vishal">
+    <preConditions onFail="MARK_RAN">
+            <tableExists tableName="patient_appointment" />
+    </preConditions>
+    <sql>
```

Comment:
use liquibase "addColumn" tag instead of SQL

Reply:
Done